test/functionalTest/cases/117_convop_using_standard_ops/getContextEntitiesByEntityId.test

```diff
@@ -0,0 +1,646 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
A thought:

Probably the new tests in 17_convop_using_standard_ops (which are designed to be exahustive) make the ones in 000_ngsi10_convenience and 000_ngsi9_convenience obsolete. However, I'd not remove the 000_*_convenience ones: the only problem in having *too many* tests is the effort to write them. Given that the effort for 000_*_convenience is already payd, let's keep them.

NTC